ci-operator/step-registry/baremetal/lab/ipi/conf/external-lb/baremetal-lab-ipi-conf-external-lb-commands.sh

```diff
@@ -15,6 +15,10 @@ trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wa
 echo "Creating patch file to enable UserManaged loadbalancer: ${SHARED_DIR}/install-config.yaml"
 
 cat > "${SHARED_DIR}/external_lb_patch_install_config.yaml" <<'EOF'
```

Comment:
```suggestion
cat > "${SHARED_DIR}/external_lb_patch_install_config.yaml" <<EOF
```

otherwise, no variable substitutions will happen in the heredoc.